mera/utils.py

```diff
@@ -0,0 +1,81 @@
+from typing import Optional
+import pandas as pd
```

Comment:
These imports need to be sorted and separated, like

```python
from typing import Optional

import numpy as np
import pandas as pd
```

However, you may not need the `Optional` import. See my comments later on.